source/getting-started/use_cases/vulnerability_detection.rst

```diff
@@ -88,8 +88,12 @@ Expand the output to see an example of a vulnerability detection alert.
 
 Vulnerability detection dashboard:
 
-.. thumbnail:: ../../images/getting_started/use_case_vulnerability.png
-   :align: center
-   :wrap_image: No
+.. thumbnail:: /images/getting_started/use-cases/wazuh-use-cases-vuln1.png
```

Comment:
This structure (one image above the other) doesn't look good. 
I prefer adding a `class ` here even though they have different sizes if there's no other way.
Also, these images need to be edited. I can see the black line from the header you removed. 
![image](https://user-images.githubusercontent.com/79285586/141326482-ab1b6275-7428-4cd1-8e2f-0f83eb2a78ad.png)